scripts/lib/libvirt/cleanup.py

```diff
@@ -66,7 +66,7 @@ def main():
     try:
         print("undefining {0}".format(net_name))
         network.undefine()
-    except libvirt.libvirtError:
+    except (libvirt.libvirtError, UnboundLocalError):
```

Comment:
So blame a real python developer. I won't invest more time. I can't do python and i just thought about low hanging fruits to avoid errors.